agent/runner/jobs/pbm_job_logger.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jobs
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/url"
+	"sync/atomic"
+	"time"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/percona/pmm/api/agentpb"
+)
+
+type pbmJob string
+
+const (
+	pbmBackupJob  pbmJob = "backup"
+	pbmRestoreJob pbmJob = "restore"
+)
```

Comment:
You can use job types instead: https://github.com/percona/pmm/blob/d7fd4bc9fa0ff08923785903ca0db574e68c07aa/agent/runner/jobs/job.go#L29-L34

Reply:
the `pbmJob` are specific events for pbm though. They are used as prefix when filtering logs (https://docs.percona.com/percona-backup-mongodb/reference/pbm-commands.html#pbm-logs)